release_notes/370-release-notes.adoc

```diff
@@ -0,0 +1,185 @@
+|GA
+|GA
+|DEP
+
+| `--rhacs` option for the `roxctl` helm output command
+| GA
+| DEP
+| DEP
+
+|Security policies without a `policyVersion`
+|DEP
+|DEP
+|REM
+
+|`/v1/policies` API endpoint response: *field* response body parameter.
+|DEP
+|DEP
+|REM
+
+|====
+
+[id="deprecated-features-370"]
+=== Deprecated features
+
+** *Anchore*, *Tenable*, and *Docker Trusted Registry* integrations: The {product-title-short} scanner supersedes these integrations.
+** *External authorization plug-in for scoped access control*: Use the existing in-product scoped access control.
+** *`FROM` option in the Disallowed Dockerfile line policy field*: Any policies containing the Disallowed Dockerfile line policy field with the `FROM` option must be updated to remove those policy sections.
+
+[id="removed-features-370"]
+=== Removed features
+
+* {product-title-short} 3.70 no longer supports security policies that do not have `policyVersion` 1.1, including (but not limited to) importing policies.
+* {product-title} will not allow deleting default policies. Rather than deleting policies, you can disable default policies that you do not need.
+* The `/v1/policies` API endpoint response will not return the *field* response body parameter.
+
+[id="bug-fixes-3670"]
+== Bug fixes
+
+* When configuring a JFrog Artifactory integration, the username and password fields are now optional to allow anonymous pulls. (ROX-10090)
+* Validation to the web user interface for endpoint URLs in the generic webhook integration caused errors. This issue was fixed. (ROX-9902)
+* The policy *OpenShift: Kubeadmin Secret Accessed* is no longer triggered if the request was from the default OpenShift `oauth-apiserver-sa` service account, because this is an expected access pattern for the OpenShift API server. (ROX-10018)
+* The ability to enable notifications for multiple policies selected in the Policies list has been reinstated. To enable notifications, select one or more policies and choose Enable notification from the Bulk Actions menu. (ROX-9985)
+* Read/write permission issue for vulnerability reports has been fixed. (ROX-9880)
+* The ability to enable notifications for multiple policies selected in the Policies list has been reinstated. To enable notifications, select one or more policies and choose *Enable notification* from the *Bulk Actions* menu. (ROX-9985)
+* Read/write permission issue for vulnerability reports has been fixed. (ROX-9880)
+* Fixed issue that caused connection problems to the {ocp} console after connecting to {product-title-short} or the inability to connect to {product-title-short} if a connection to the {ocp} console existed. Central will now respond with a 421 Misdirected Request status code to requests where the `ServerName` sent via TLS SNI does not match the `:authority` (Host) header. This feature can be turned off by setting the environment variable `ROX_ALLOW_MISDIRECTED_REQUESTS=true`. (ROX-9625)
```

Comment:
For 'sent via TLS', the IBM Style Guide - Word usage section has guidance on whether 'via' is accepted or if it should be replaced with 'using, etc.) I think you are OK here, but PTAL

Reply:
Oh, good suggestion - I checked and since TLS is a transport mechanism, I think via is OK here too.